### ISSUE TITLE
why did you use kaiming_uniform to init parameters?

### ISSUE THREAD

**Issue by JingsenZhang** [guoyang9/NCF#17]
nn.init.kaiming_uniform_(self.predict_layer.weight, a=1, nonlinearity='sigmoid')

can you tell me why?
thank you!

**Comment by guoyang9** [comment]
Xiangnan leverages [lecun_uniform](https://github.com/hexiangnan/neural_collaborative_filtering/blob/4aab159e81c44b062c091bdaed0ab54ac632371f/NeuMF.py#L102) to initialize the prediction layer weights, which is absent in pytorch.  So I took the `kaiming_uniform` to replace it due to the identical form with the `lecun_uniform` in Keras.
